resources/packs/terraform/tfstate.go

```diff
@@ -138,7 +138,7 @@ func (t *mqlTerraformState) GetResources() (interface{}, error) {
 		return nil, nil
 	}
 
-	// resolve all tfstate resources, to achive this we need to walk all modules
+	// resolve all tfstate resources, to archive this we need to walk all modules
```

Comment:
I think, "achieve" is meant in the original text.

Reply:
Fixed